java/src/jmri/jmrix/openlcb/OlcbPowerManager.java

```diff
@@ -0,0 +1,98 @@
+package jmri.jmrix.openlcb;
+
+import jmri.PowerManager;
+import jmri.managers.AbstractPowerManager;
+import jmri.jmrix.can.CanSystemConnectionMemo;
+
+import org.openlcb.EventID;
+import org.openlcb.OlcbInterface;
+import org.openlcb.implementations.BitProducerConsumer;
+import org.openlcb.implementations.EventTable;
+import org.openlcb.implementations.VersionedValueListener;
+
+/**
+ * Extend jmri.managers.AbstractPowerManager for OpenLCB controls.
+ *
+ * @author Bob Jacobsen Copyright (C) 2025
+ */
+public final class OlcbPowerManager extends AbstractPowerManager<CanSystemConnectionMemo> {
+
+    OlcbAddress addrOn  = new OlcbAddress(new EventID("01.00.00.00.00.00.FF.FE"));
+    OlcbAddress addrOff = new OlcbAddress(new EventID("01.00.00.00.00.00.FF.FF"));
+
+    private final OlcbInterface iface;
+
+    VersionedValueListener<Boolean> powerListener;
+    BitProducerConsumer pc;
+    EventTable.EventTableEntryHolder onEventTableEntryHolder = null;
+    EventTable.EventTableEntryHolder offEventTableEntryHolder = null;
+    private static final int PC_DEFAULT_FLAGS = BitProducerConsumer.DEFAULT_FLAGS &
+            (~BitProducerConsumer.LISTEN_INVALID_STATE);
+
+    public OlcbPowerManager(CanSystemConnectionMemo memo) {
+        super(memo);
+        this.iface = memo.get(OlcbInterface.class);
+
+        int flags = PC_DEFAULT_FLAGS;
+        log.debug("Power Manager Flags: default {} overridden {} listen bit {}", PC_DEFAULT_FLAGS, flags,
+                    BitProducerConsumer.LISTEN_EVENT_IDENTIFIED);
+        dispose();
```

Comment:
Having dispose() in the constructor looks very strange. I recommend adding a comment telling why it's there.